pkg/csource/csource.go

```diff
@@ -501,8 +502,55 @@ func (ctx *context) copyout(w *bytes.Buffer, call prog.ExecCall, resCopyout bool
 	}
 }
 
+func (ctx *context) factorizeAsFlags(value uint64, flags []string) ([]string, uint64) {
+	if len(flags) == 0 || value == 0 {
+		return nil, value
+	}
+
+	currentFlag := flags[0]
+	subset, remainder := ctx.factorizeAsFlags(value, flags[1:])
```

Comment:
Well, that's exponential on the number of flags defined for that type but in practice that number should always be quite low and since most values to factorize are sparse, we should prune most of the branches of the exploration tree as well. Do you think it's worth improving upon ?

Reply:
I'd still add some cut off to the maximum number of combinations we try here.